### DISCUSSION TITLE
Support of Microsoft Windows Server 2025

### DISCUSSION THREAD

**Discussion by mjschwaiger** [adoptium/adoptium-support#1193] [Q&A] [answered]
Microsoft released Windows Server 2025 on Nov 1st, 2024 and the official build number is 26100.
https://learn.microsoft.com/en-us/windows-server/get-started/windows-server-release-info

Changes for the detection of Windows Server 2025 have already been added to openjdk:
https://bugs.openjdk.org/browse/JDK-8340387

Currently supported platforms:
https://adoptium.net/supported-platforms/

Is there a roadmap for when the newly released Microsoft Windows Server 2025 will be included in the test environment and when it will be part of the supported platforms?

Please keep us updated. Thanks in advance.

**Comment by karianna** [comment]
Have added this to the PMC agenda

**Reply by r-shikimi** [reply]
Is this discussion being addressed at the PMC Meeting?
At the PMC Meeting, we check for Issues with the "PMC-agenda" label. Since this is a Discussion, is it being overlooked?

**Reply by jerboaa** [reply]
That's probably right. I've manually added it to be discussed on today's call. Next time, an issue would be better than a discussion item. Sorry about that and thanks for the reminder.

**Reply by S-Callier** [reply]
Hi!

I was wondering if Windows Server 2025 support was discussed during your recent PMC meeting.
If so, is it realistically possible for it to be included as a supported platform in the upcoming release on 7/15, or is it more likely to come in a later release?

Thank you for all your hard work on Temurin!

**Reply by karianna** [reply, marked as answer]
From PMC Minutes: Natural progression for Windows Server version update.
Infra and test will ensure there is sufficient in place for release workload in Jenkins.

I don't have an ETA on this, but is was approved in principle

**Reply by S-Callier** [reply]
Thank you very much.

**Reply by mjschwaiger** [reply]
Thank you for considering this new Windows Server version in an upcoming Adoptium release.

So please note that active support for Windows Server 2022 will end in October 2026 (just over a year to go).

Therefore, our customers are also asking us to support Windows Server 2025 as soon as possible.

Source: https://endoflife.date/windows-server

**Reply by mjschwaiger** [reply]
As of right now, the Adoptium site already says that Windows Server 2025 is supported 
![2025-06-05-120313-supported-platforms-adoptium-windows64](https://github.com/user-attachments/assets/da7c2e21-4518-43fb-9719-72d93728c44d)
Source: https://adoptium.net/supported-platforms/

**Reply by mjschwaiger** [reply]
The change to the documentation regarding 'Windows Server 2025' was included in the following commit:
https://github.com/adoptium/adoptium.net/pull/1191/files#diff-4e38f2085e62f0fe463cf5f95d4ce893960be264817f353f1a88a1aa7d73d93fR7

However, no further notice or explanation was provided as to why and how this support was suddenly included.
Are there any other issues that should be linked?

**Reply by karianna** [reply]
https://github.com/adoptium/containers/pull/707 is a related issue.

**Reply by smlambert** [reply]
We now also have Windows Server 2025 nodes available for testing on, which is the other reason we are able to state it as a supported platform.

I will now remove the PMC-Agenda tag from this and assume the discussion has run its full course.

**Comment by gdams** [comment]
This can be resolved now. All outstanding action items have been resolved
